app/components/provider_interface/user_org_permission_breakdown_component.rb

```diff
@@ -0,0 +1,46 @@
+module ProviderInterface
+  class UserOrgPermissionBreakdownComponent < ViewComponent::Base
```

Comment:
Could we rename this to make it clear that it is not affected by user level permissions? This component is displaying a list of partner organisations for which the given provider has and does not have the given permission. Maybe something like `ProviderPartnerPermissionBreakdownComponent`